src/MetadataUtility/Metadata/Predicates.cs

```diff
@@ -13,7 +13,7 @@ public static class Predicates
         target => !string.IsNullOrWhiteSpace(target.FileSystem.Path.GetFileName(target.Path));
 
     public static readonly Func<TargetInformation, bool> IsFlacFile =
-        target => Flac.IsFlacFile(target.FileStream).IfFail(false);
+        target => Flac.IsValidFlacFile(target.FileStream).IfFail(false);
```

Comment:
If you keep the name, update the wrapper extension method to reflect what is is actually doing (`IsFlacFile` -> `IsValidFlacFile`)

But (see previous comments) I see value in granular predicates.